content/docs/05.developer-guide/02.tasks.md

```diff
@@ -253,6 +253,53 @@ You cannot access the output of a sibling task as tasks will be run in parallel.
 ::
 
 
+### Dag
```

Comment:
```suggestion
### DAG
```
This is an abbreviation... decide  what you prefer, I'd prefer DAG over Dag

Reply:
Agreed